### ISSUE TITLE
Base OS security updates

### ISSUE THREAD

**Issue by TheNetworkIsDown** [very-undude/ultimatedeployment#16]
There should be a way to update the Base OS (CentOS 7) to the most recent patch levels.
Not sure if there is just no mention about it, or if it is not mentioned for a reason.

**Comment by very-undude** [comment]
Good idea, there's no reason it is not mentioned/available, just too little time and too many things to do!
